src/main/scala/org/labrad/Server.scala

```diff
@@ -262,13 +264,58 @@ object Server {
    * Includes basic parsing of command line options.
    */
   def run(s: Server[_, _], args: Array[String]): Unit = {
-    val options = Util.parseArgs(args, Seq("name", "host", "port", "password"))
+    val parser = new ArgotParser("labrad",
```

Comment:
I was thinking that it makes it easier to test really. defineParameters is bad name - maybe parseParameters is better - it just returns the parameters needed to run the server. It's then possible to test parameter parsing without needing to start a server (though a mock server can be used).